test/markdownlint-test-result-object.js

```diff
@@ -346,7 +346,13 @@ test.cb("resultFormattingV3", (t) => {
         "# Heading   \n" +
         "\n" +
         "Text\ttext\t\ttext\n" +
-        "Text * emphasis * text"
+        "Text * emphasis * text\n" +
+        "\t\tInformation"
```

Comment:
Is there a reason to test it here? If not, it would be better (also easier) to add a new test file and use the "{MD010}" syntax. I'd want one for some number (4?) and another for 0. There should already be a set of files that verify the fix for this rule.

Reply:
> Is there a reason to test it here?

It was one of the tests on MD010, so I decided to use it.

> If not, it would be better (also easier) to add a new test file

Could you point to some examples? Even though I found multiple test files, I was not able understand where it was being tested.

> use the "{MD010}" syntax

What is this syntax? How would I tell the expected fixes using this syntax?

Reply:
Have a look at this set of files:
- https://github.com/DavidAnson/markdownlint/blob/main/test/detailed-results-html-tags.md
- https://github.com/DavidAnson/markdownlint/blob/main/test/detailed-results-html-tags.json
- https://github.com/DavidAnson/markdownlint/blob/main/test/detailed-results-html-tags.results.json
- https://github.com/DavidAnson/markdownlint/blob/main/test/detailed-results-html-tags.md.fixed

I prefer that style over changing code - the test you changed here is mainly about verifying the result object format - not rule behavior and fixes like the above format.

(My comment about "{MD010}" does not apply here, only to tests that do not verify fix output, sorry.)

Reply:
Also, let's test for the default (1), a typical value (4) and the edge case (0).

Reply:
434edc9